packages/flame/lib/src/components/text_box_component.dart

```diff
@@ -93,9 +112,13 @@ class TextBoxComponent<T extends TextRenderer> extends PositionComponent {
 
   bool get finished => _lifeTime > totalCharTime + _boxConfig.dismissDelay;
 
+  int get _actualTextLength {
+    return _lines.map((e) => e.length).fold(0, (p, c) => p + c);
```

Comment:
`reduce` can be used

Reply:
I didn't want it to explode if someone created an empty text